sst/test/src/tg4_request_misaligned.cc

```diff
@@ -0,0 +1,424 @@
+#include "sstsimengtest.hh"
+
+TEST_GROUP(TG4, "SSTSimEng_handles_misaligned_memory_requests",
```

Comment:
Why isn't 8-bit cache line tested in this group, where as in request_split tests a cache line width of 8-bits is used?

Reply:
The reason there is no 8-bit group is that addresses are always byte aligned, so that means each incremental address will be a separate cache line. i.e `ld x5 #2` and `ld x5 #3` will be on cache line 2 (Start Addr: 2*8 = 16) and 3 (Start Addr: 3*8 = 24) respectively. This means that all load and store instructions will start at the start of a cache line. Cases when requests are larger than 8-bit cache line are already handled in request split.

Reply:
Understood. Thanks